### PR TITLE
Add new page with organizer reports

### PR DESCRIPTION
**Pull request by nishanthvijayan**
As suggested by @differentreality here https://github.com/openSUSE/osem/pull/1114#issuecomment-241228744 this PR adds a new admin side bar item 'Reports'
that displays
- All events and what they are missing (commercial, difficulty level, track, subtitle etc)
- List of events that have no commercials
- List of events where speakers had requirements (event.description)  
  Solves issue #1155 

all_events
commercials
speaker_requirements

### CODE REVIEW

**Comment by nishanthvijayan** [comment]
@differentreality I have not added a missing speakers table yet.

> Missing speakers, people who have not checked in but have a talk
How do we check if someone 'checked in'?  using attended in registration?

**Comment by differentreality** [comment]
If they are marked as attended (ie. if they are registered and if their registration.attended attribute is true)

File: app/views/layouts/_admin_sidebar.html.haml [hunk 1]
```diff
@@ -72,6 +72,8 @@
           - if can? :update, @conference.program.events.build
             %li{:class=> active_nav_li(admin_conference_program_events_path(@conference.short_title))}
               = link_to 'Events', admin_conference_program_events_path(@conference.short_title)
+            %li{:class=> active_nav_li(reports_admin_conference_program_path(@conference.short_title))}
```
Comment (differentreality) [inline app/views/layouts/_admin_sidebar.html.haml]: I'd move that at the end of the list

**Comment by differentreality** [comment]
@hennevogel any thoughts about this?

**Comment by differentreality** [comment]
What about merging this as it is and adding the suggestions of the previous comment later on?

**Pull request merged**